docs/samples/browser-plugin-meetings/app.js

```diff
@@ -955,6 +953,38 @@ const localMedia = {
   }
 }
 
+function handleStreamPublishedState(meeting) {
```

Comment:
should this be `handleStreamUnpublishedState()`?

Reply:
The event is 'published-state-update' 

So the `publishedState` can mean both published and unpublished. So, this name should be fine